front-end/src/ui-components/Form.tsx

```diff
@@ -23,10 +23,10 @@ export function Form({ standalone=true, ...props } : FormProps): React.ReactElem
 }
 
 const StyledForm = styled(SUIForm)`
-	background-color: ${(): string => _standalone ? '#FFF' : 'default'};
+	background-color: ${(): string => _standalone ? 'white' : 'default'};
 	padding: ${(): string => _standalone ? '2rem 3rem 3rem 3rem' : '0'};
 	margin-top: ${(): string => _standalone ? ' 4rem' : '0'};
-	border: ${(): string => _standalone ? '1px solid #EEE' : 'none'};
+	border: ${(): string => _standalone ? '1px solid #EBF0F5' : 'none'};
```

Comment:
It's verbose, but you can use it there, over 3 lines isn't it?

Reply:
Sure 👍(also initially thought it's a bit verbose )

Reply:
Actually, getting an 'invalid property' warning here